### PR TITLE
fix(Embed): do not show control when not appropriate

### PR DESCRIPTION
**Pull request by lucivpav**
This PR resolves issue #1767, which states that:
> the pause control button is visible, even if we are using an iframe in Embed component.

**Changes**
* control button is not shown when not appropriate
* background set to <del>`grey`</del> `colorScheme.brand.background`
* added YouTube example
* changed icon from Play to Search, on the example with map

### CODE REVIEW

File: docs/src/examples/components/Embed/Types/index.tsx [hunk 1]
```diff
@@ -10,6 +10,16 @@ const Types = () => (
...
       examplePath="components/Embed/Types/EmbedExample"
     />
+    <ComponentExample
+      title="IFrame"
+      description="An embed component can embed any iframe."
+      examplePath="components/Embed/Types/EmbedExampleIframe"
+    />
+    <ComponentExample
+      title="YouTube"
+      description="An embed component can embed a YouTube video by iframe."
```
Comment (layershifter) [inline docs/src/examples/components/Embed/Types/index.tsx:10]: Let's move this under `Usage` section.
And keep `Iframe` under `Slots`

File: docs/src/examples/components/Embed/Types/EmbedExampleIframe.shorthand.tsx [hunk 2]
```diff
@@ -10,6 +10,7 @@ const EmbedExampleIframe = () => (
       height: 450,
       width: 600,
     }}
+    control={{ name: 'search' }}
```
Comment (layershifter) [inline docs/src/examples/components/Embed/Types/EmbedExampleIframe.shorthand.tsx]:
```suggestion
    control='search'
```
nit: can be just icon's name

File: packages/react/src/themes/teams/components/Embed/embedStyles.ts [hunk 3]
```diff
@@ -13,6 +13,7 @@ export default {
       width: v.width,
       height: v.height || 'auto',
       outline: 0,
+      background: 'grey',
```
Comment (layershifter) [inline packages/react/src/themes/teams/components/Embed/embedStyles.ts]: Let's create a variable instead and use some color from color schema

File: docs/src/examples/components/Embed/Types/index.tsx (hunk 1 above)
Comment (lucivpav) [reply docs/src/examples/components/Embed/Types/index.tsx:10]: Both IFrame example and video example belong under Slots section right?

Comment (layershifter) [reply docs/src/examples/components/Embed/Types/index.tsx:10]: Yep, no `Types`. It doesn't have sense there

**Comment by lucivpav** [comment]
These changes will be integrated in #1758 instead.

**Pull request closed without merging**